stdlib/public/SwiftShims/Visibility.h

```diff
@@ -91,9 +91,9 @@
 # define SWIFT_EXPORT_ATTRIBUTE __attribute__((__visibility__("default")))
 #endif
 
-#else
+#else  // FIXME: this #else should be some sort of #elif Windows
```

Comment:
Unfortunately, there is no file format indicator for COFF.  That really should be a check for COFF.  !MachO, !ELF currently implies COFF. At some point, it will require a !WASM check.